finetune/base_models/bert/modeling.py

```diff
@@ -246,8 +246,14 @@ def __init__(
             with tf.compat.v1.variable_scope("pooler"):
                 # We "pool" the model by simply taking the hidden state corresponding
                 # to the first token. We assume that this has been pre-trained
-
-                first_token_tensor = tf.squeeze(self.sequence_output[:, 0:1, :], axis=1)
+                output_shape = tf.shape(self.sequence_output)
```

Comment:
This is done because there's some small chance we generate a chunk w/ no valid tokens?

Reply:
Yeah, with featurizer fusion theres chance that we pass 0 length chunks to the featurizer. Theres a chance we can avoid this with by checking for the invalid chunks in the featurizer fusion function, but I wasn't able to get it to work before. Will give it another shot.

Reply:
Unfortunately a bit more complicated then expected, as there are a lot of values returned by the featurizer that vary from base model to base model, and we have to replicate them all if we want to write a conditional in featurizer_fusion(). Won't be able to get it done today, but someone might want to take a look at a later point.